test/batch/configuration.ts.snap

```diff
@@ -0,0 +1,175 @@
+      path: [Array],
+      type: 'any.required',
+      context: [Object]
+    }
+  ]
+}",
+]
+`;
+
+exports[`Configuration Configuration is invalid if condition is missing 1`] = `
+Array [
+  "ERROR:  Configuration file is invalid",
+  "[Error [ValidationError]: \\"rules[0].condition\\" is required] {
+  _original: { rules: [ [Object] ] },
+  details: [
+    {
+      message: '\\"rules[0].condition\\" is required',
+      path: [Array],
+      type: 'any.required',
+      context: [Object]
+    }
+  ]
+}",
+]
+`;
+
+exports[`Configuration Configuration is invalid if locksReviewTeam is empty or missing 1`] = `
+Array [
+  "ERROR:  Locks Review Team should be provided",
+]
+`;
+
+exports[`Configuration Configuration is invalid if min_approvals is less than 1 1`] = `
+Array [
+  "ERROR:  Configuration file is invalid",
+  "[Error [ValidationError]: \\"rules[0].min_approvals\\" must be greater than or equal to 1] {
+  _original: { rules: [ [Object] ] },
+  details: [
+    {
+      message: '\\"rules[0].min_approvals\\" must be greater than or equal to 1',
+      path: [Array],
+      type: 'number.min',
+      context: [Object]
+    }
+  ]
+}",
+]
+`;
+
+exports[`Configuration Configuration is invalid if min_approvals is less than 1 2`] = `
+Array [
+  "ERROR:  Configuration file is invalid",
+  "[Error [ValidationError]: \\"rules[0].min_approvals\\" must be greater than or equal to 1] {
+  _original: { rules: [ [Object] ] },
+  details: [
+    {
+      message: '\\"rules[0].min_approvals\\" must be greater than or equal to 1',
+      path: [Array],
+      type: 'number.min',
+      context: [Object]
+    }
+  ]
+}",
+]
+`;
+
+exports[`Configuration Configuration is invalid if min_approvals is missing 1`] = `
+Array [
+  "ERROR:  Configuration file is invalid",
+  "[Error [ValidationError]: \\"rules[0].name\\" must be a string] {
+  _original: { rules: [ [Object] ] },
+  details: [
+    {
+      message: '\\"rules[0].name\\" must be a string',
+      path: [Array],
+      type: 'string.base',
+      context: [Object]
+    }
+  ]
+}",
+]
+`;
+
+exports[`Configuration Configuration is invalid if name is missing 1`] = `
+Array [
+  "ERROR:  Configuration file is invalid",
+  "[Error [ValidationError]: \\"rules[0].name\\" is required] {
+  _original: { rules: [ [Object] ] },
+  details: [
+    {
+      message: '\\"rules[0].name\\" is required',
+      path: [Array],
+      type: 'any.required',
+      context: [Object]
+    }
+  ]
+}",
+]
+`;
+
+exports[`Configuration Configuration is invalid if teamLeadsTeam is empty or missing 1`] = `
+Array [
+  "ERROR:  Team Leads Team should be provided",
+]
+`;
+
+exports[`Configuration Rule kind AndRule does not allow invalid field any 1`] = `
+Array [
+  "ERROR:  Rule \\"condition\\" was expected to be of kind \\"AndRule\\" because it had the field \\"all\\", but it also has the field \\"any\\", which belongs to another kind. Mixing fields from different kinds of rules is not allowed.",
```

Comment:
GitHub doesn't show nor wraps line this long
![image](https://user-images.githubusercontent.com/17856421/149993271-a4626deb-617d-49c7-96c8-0fbfe99d1984.png)